app/assets/javascripts/course/components/AudioComponent.js

```diff
@@ -24,6 +24,13 @@ class AudioComponent extends React.Component {
     this.onDuration = this.onDuration.bind(this)
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      playing: false,
+      progress: 0,
+    })
+  }
```

Comment:
Generally it's good to explicitly do a comparison to see if the props have changed (`this.props != nextProps`) to make sure that you are only resetting the state of the audio player when a new component is rendered.